client/src/SignUpShopper.js

```diff
@@ -0,0 +1,98 @@
+import { useEffect, useState } from 'react';
+import { useHistory, useParams } from 'react-router-dom';
+import Api from './Api';
+
+//to add new component, make new file
+//to create (crud), use forms
+function SignUpShopper() {
+
+    //:id from Sections.js 
+    const {id} = useParams();
```

Comment:
In the case of User, we don't need to necessarily pass the user's id around in the route path (and it's not configured in your route to do so). The reason is that the user will be "logged in" and we can automatically detect that in the server and do operations on that user. So I don't think we'll need this in THIS CASE,